lang_python.adoc

```diff
@@ -114,30 +130,33 @@ https://setuptools.readthedocs.io/en/latest/[Setuptools] is the de-facto
 standard way of building and installing python packages. This library
 allows extension with
 https://setuptools.readthedocs.io/en/latest/setuptools.html#extending-and-reusing-setuptools[plugins].
-https://github.com/KOLANICH have developed
-https://github.com/KOLANICH/kaitaiStructCompile.py[an extension to
+https://gitlab.com/KOLANICH has developed
+https://gitlab.com/kaitaiStructCompile.py/kaitaiStructCompile.setuptools[an extension to
 setuptools], allowing you to automate compilation of `*.ksy` files as the
 part of python module build process. The library provides a mostly
 declarative syntax to facilitate installation: you add a parameter
 `kaitai` to your `setup.py` where you enumerate files that you want
 compiled, output directory, compilation flags, and source code
 post-compilation transformations. Here are some examples:
-https://github.com/KOLANICH/NTMDTRead/blob/master/setup.py[1],
-https://github.com/KOLANICH/SpecprParser.py/blob/master/setup.py[2].
+https://gitlab.com/KOLANICH/NTMDTRead/blob/master/setup.py[1],
+https://gitlab.com/KOLANICH/SpecprParser.py/blob/master/setup.py[2],
+https://gitlab.com/KOLANICH/FrozenTable.py/blob/master/setup.py[3],
+https://gitlab.com/KOLANICH/MFCTool.py/blob/master/setup.py[4],
+https://gitlab.com/KOLANICH/RDataParser.py/blob/master/setup.py[5].
 
 Here are
-https://github.com/KOLANICH/kaitaiStructCompile.py/blob/master/kaitaiStructCompile/config.schema.json[the
-docs] in JSON Schema format.
+https://gitlab.com/kaitaiStructCompile.py/kaitaiStructCompile.schemas/blob/master/kaitaiStructCompile/schemas/schemas/config.schema.json[the
+docs] in https://json-schema.org/specification.html[JSON Schema] format.
 
 Please remember that this lib is VERY UNSTABLE and very likely to be
 improved in a way that changes API (but due to API simplicity it'd be
 easy to fix your code), but I guess it's much better to use this than to
 reinvent the wheel.
-https://github.com/KOLANICH/kaitaiStructCompile.py/issues[Contributions
+https://gitlab.com/kaitaiStructCompile.py/kaitaiStructCompile.py/issues[Contributions
 are welcome!]
 
-Please note that this lib is NOT available on
-https://pypi.python.org[pypi] and if you want to install it
-automatically, you should add
-`git+https://github.com/KOLANICH/kaitaiStructCompile.py` into
-`dependency_links` in the config passed to `setuptools.setup`.
+Please note that this lib is NOT available on https://pypi.python.org[pypi]
+and if you want to install it automatically, you should add
+`kaitaiStructCompile.setuptools @ https://gitlab.com/kaitaiStructCompile.py/kaitaiStructCompile.setuptools`
+into `setup_requires` into `options` section of your `setup.cfg` and into
+`requires` of `build_system` section of your `pyproject.toml`.
```

Comment:
Uh, this is _way too many_ machine identifiers in one sentence, I've been lost a long time ago, it would be better to directly show the line ``kaitaiStructCompile.setuptools @ https://gitlab.com/kaitaiStructCompile.py/kaitaiStructCompile.setuptools` in action, on the same place where it should be in practice (i.e. use a code block with a minimal samples of the resulting files after the additions).